backend/app/api/routes/question.py

```
@@ -1,7 +1,7 @@
 import csv
 from datetime import datetime
 from io import StringIO
-from typing import Any
+from typing import Any ,cast
```

Ruff (0.11.9)

`typing.cast` imported but unused

Remove unused import: `typing.cast`

(F401)